### ISSUE TITLE
Setting attention mask in bert-large-uncased-whole-word-masking-finetuned-squad

### ISSUE THREAD

**Issue by svanschalkwyk** [explosion/spaCy#5047]
Trying to use SpaCy for SQUAD. 
```
name = "bert-large-uncased-whole-word-masking-finetuned-squad"
nlp = TransformersLanguage(trf_name=name, meta={"lang": "en"})
nlp.add_pipe(nlp.create_pipe("sentencizer"))
nlp.add_pipe(TransformersWordPiecer.from_pretrained(nlp.vocab, path))
nlp.add_pipe(TransformersTok2Vec.from_pretrained(nlp.vocab, path))
```
qa = nlp(question....
[and I need help further down as well, i.e. logits]
Any help would be greatly appreciated.
